docs/maintaining-a-track.md

```diff
@@ -0,0 +1,79 @@
+# Maintaining an Exercism Language Track
+
+* [Philosophy](#philosophy)
+
+  * [Be Kind](#be-kind)
+  * [Be Responsive](#be-responsive)
+  * [Empower New Contributors](#empower-new-contributors)
+  * [Recognize Great Contributions](#recognize-great-contributions)
+  * [Improve the Learning Experience](#improve-the-learning-experience)
+  * [Collaborate](#collaborate)
+
+* [Day-to-Day Tactics](#day-to-day-tactics)
+
+  * [Watch Key Repositories](#watch-key-repositories)
+  * [Perform Common Tasks](#perform-common-tasks)
+
+## Philosophy
+
+### Be Kind
+
+First and foremost, strive always to be kind, humble and show gratitude for the work being contributed.
```

Comment:
It depends on which you would like someone to do. Should you always strive or always to be kind?  Strive always to be kind, has more weight to the kindness.  Otherwise, always strives holds less impetus to be kind, but to always strive.